docs/layers/monitoring/datadog/datadog.mdx

```diff
@@ -6,10 +6,24 @@ description: Monitor everything with Datadog
 ---
 import Intro from '@site/src/components/Intro';
 import KeyPoints from '@site/src/components/KeyPoints';
+import Steps from '@site/src/components/Steps';
+import Step from '@site/src/components/Step';
+import StepNumber from '@site/src/components/StepNumber';
+import Admonition from '@theme/Admonition';
+
 
 Cloud Posse advises customers that all have <i>very similar monitoring requirements</i> because we filter for customers who match our delivery model. The actual fine-tuning of it will be specific to the customer.
 
-Our goal with this ADR is to identify the reusable, standard SLI/SLO for our customers that we can readily implement time and time again.
+Our goal with this document is to identify the reusable, standard SLI/SLO for our customers that we can readily implement time and time again.
+
+<Admonition type="tip">
+  <br/>
+  This document goes very deep into the theory and practice of monitoring with Datadog.
+  <br/>
+  If you are looking for a quick start guide, see [our Setup Guide for DataDog](/layers/monitoring/datadog/setup/).
+  <br/>
```

Comment:
We shoudln't use <br/> tags, and instead adjust CSS.

Reply:
In this case, the formatting of admonitions is deliberate, to conserve space.

Reply:
I think that looks worse, but will remove `<br/>`

Reply:
That's a separate converstation, on how we should style admonitions. We shouldn't style individual admonitions. =)